app/models/version.rb

```diff
@@ -3,8 +3,11 @@
 class Version < ApplicationRecord
   include Releaseable
 
-  validates_presence_of :project_id, :number
-  validates_uniqueness_of :number, scope: :project_id
+  STATUSES = ['Deprecated', 'Removed'].freeze
```

Comment:
Assuming you checked these? (After the convo in slack) :D

Reply:
Yeah, these are the values for the new column, but I did check the `Project#version` values in the db.